contracts/ExecutionManager.sol

```diff
@@ -90,10 +118,23 @@ contract ExecutionManager is IExecutionManager, AccessControlExtended {
      * @return Minimum fees required for executing the transaction
      */
     function getMinFees(
-        uint256,
+        uint256 gasLimit_,
+        uint256 msgValue_,
+        uint256 payloadSize_,
+        bytes32 extraParams_,
         uint32 siblingChainSlug_
     ) external view override returns (uint256) {
-        return executionFees[siblingChainSlug_];
+        if (msgValue_ == 0) return executionFees[siblingChainSlug_];
+
```

Comment:
min value check is there so zero check not needed

Reply:
I kept the 0 check for the case if there is no msgValue, we will just return the execution Fees without any calculation. But if there is some value, we will check min and max values.

Reply:
cool makes sense